static/app/components/sidebar/sidebarDropdown/switchOrganization.tsx

```diff
@@ -131,10 +133,7 @@ function SwitchOrganization({organizations, canCreateOrganization}: Props) {
   );
 }
 
-const SwitchOrganizationContainer = withOrganizations(SwitchOrganization);
```

Comment:
In similar spirit, should we just have a useOrganizations counterpart for this HOC?